read-me.txt

```diff
@@ -6,3 +6,4 @@ Added for change3
 Created remote repo. Pushed it. This will be first commit after linking local to remote change4
 Editing this change in the github browser remotely - to test pull command
 After adding ak-c as collaborator. Changes added by ak-c change5
+Created feature-branch1 : added initial changes bc:1 c6
```

Comment:
change the verbiage initial changes to initial-changes

Reply:
Ok will make changes and commit

Reply:
Looks good now accepting those changes